test/representers/api/auth/feed_representer_test.rb

```diff
@@ -27,7 +27,7 @@
   end
 
   it 'has a feed rss link' do
-    _(json['_links']['prx:private-feed']['href']).must_equal feed.published_url
+    _(json['_links']['prx:private-feed']['href']).must_equal "#{feed.published_url}{?auth}"
```

Comment:
Yeah, digging down a bit, it looks like i made `feeds.private` default to `true` in the db.  So this href is now something like `https://p.prxu.org/jjgo/myfeed5/feed-rss.xml{?auth}{?auth}` in this test.